### ISSUE TITLE
Problem When Trying to Get/Install Grub (macOS)

### ISSUE THREAD

**Issue by ava-cassiopeia** [intermezzOS/intermezzOS.github.io#35]
When installing using the install file outlined [here](http://intermezzos.github.io/book/appendix/osx-install.html), and then later trying to run the `grub-mkrescue` from [here](http://intermezzos.github.io/book/making-an-iso.html) on macOS (Sierra), the `grub-mkrescue` program does not seem to come as a part of the installation, as far as I can tell. That is, nowhere in `~/opt` or `~/src` does `grub-mkrescue` show up.

I ended up just switching to Linux for the remainder of the book, but I'd still like to develop on my Mac if possible. Did I miss something, or was it installed in another area?

**Comment by steveklabnik** [comment]
So I _just_ did this with @ashleygwilliams , and she has

``` bash
$ which grub-mkrescue
/Users/../opt/bin/grub-mkresque
```

So, I wonder what happened here. The script completed? `grub` is the last bit. I also wonder if

```
if [ ! -d "grub" ]; then
```

shouldn't be `grub-mkrescue` instead...

**Comment by ava-cassiopeia** [comment]
Alright, upon closer inspection of the log, during the grub installation I get the following:

```
checking which extra warnings work... 
checking if compiling with clang... yes
checking for options to compile assembly... configure: error: could not compile assembly
```

And then exits with code `1`. I saw this the first time, but upon re-running it every time after that, the program exited with code `0`, precisely because of the 

```
if [ ! -d "grub" ]; then
```

line you mentioned. Googling that "could not compile assembly" error doesn't yield basically any result, so this might be specific to my machine.

**Comment by SamTebbs33** [comment]
@aeolingamenfel Did you ever resolve the "could not compile assembly" issue? I'm facing it now

**Comment by ava-cassiopeia** [comment]
@SamTebbs33 No, sorry. It's bee awhile since I've taken a look at this though. I might try to see if I can still replicate this issue here later.

**Comment by steveklabnik** [comment]
Note that the second edition version needs no assembly and is FAR easier to get going on macOS, you may want to try that.

**Comment by SamTebbs33** [comment]
As @steveklabnik mentioned, installing Grub 2 should work on most macOS systems. I just followed this manual and it worked for me: https://www.gnu.org/software/grub/manual/grub/grub.html#Obtaining-and-Building-GRUB
